server/apiv2/handlers/keyspace.go

```diff
@@ -230,14 +234,16 @@ type UpdateConfigParams struct {
 // UpdateKeyspaceConfig updates target keyspace's config.
 // This api uses PATCH semantic and supports JSON Merge Patch.
 // format and processing rules.
+//
 // @Tags     keyspaces
 // @Summary  Update keyspace config.
-// @Param    name  path  string              true  "Keyspace Name"
+// @Param    name  path  string                        true  "Keyspace Name"
```

Comment:
ditto

Reply:
Updated, cc @rleungx  I format it with `swag fmt`.